integration/apigateway-domain/index.ts

```diff
@@ -0,0 +1,80 @@
+import * as aws from '@pulumi/aws';
+import * as pulumicdk from '@pulumi/cdk';
+import { RestApi } from './rest-api';
+import { ResourceAttributeMappingArray, ResourceMapping } from '../../lib/interop';
+
+class ApiGatewayStack extends pulumicdk.Stack {
+    constructor(id: string, options?: pulumicdk.StackOptions) {
+        super(id, options);
+        this.node.setContext('@aws-cdk/aws-apigateway:disableCloudWatchRole', 'true');
+
+        new RestApi(this, 'test-api');
+        this.synth();
+    }
+}
+
+const stack = new ApiGatewayStack('test-api-domain', {
+    remapCloudControlResource: (logicalId, typeName, props, options): ResourceMapping | undefined => {
```

Comment:
We're keeping this option for GA? This is interesting, I need to read up on this.